common/tests/refcount_ut/refcount_ut.c

```diff
@@ -292,6 +292,7 @@ BEGIN_TEST_SUITE(TEST_SUITE_NAME_FROM_CMAKE)
         POS_HANDLE p, clone_of_p;
         p = Pos_Create(2);
         clone_of_p = Pos_Clone(p);
+        (void)clone_of_p;

```

Comment:
>     (void)clone_of_p; [](http://example.com/codeflow?start=3&length=22)

I think that one of the 2 calls below should be to destroy clone_of_p

Reply:
valgrind says close_of_p should not be destroyed.